teams/teams-ps/teams/Get-LicenseReportForChangeNotificationSubscription

```diff
@@ -0,0 +1,55 @@
+---
+external help file: Microsoft.TeamsCmdlets.PowerShell.Custom.dll-Help.xml
+Module Name: MicrosoftTeams
+title: Get-LicenseReportForChangeNotificationSubscription
+author: anandab
+ms.author: anandab
+manager: alagra
+online version: https://docs.microsoft.com/powershell/module/teams/get-licensereportforchangenotificationsubscription 
+schema: 2.2.0-preview
+---
+
+# Get-LicenseReportForChangeNotificationSubscription
+
+## SYNOPSIS
+
+This cmdlet tells whether a user has the required license to export their messages via [change notification subscription](https://docs.microsoft.com/en-us/graph/teams-licenses)
+
+## SYNTAX
+
+```
+Get-LicenseReportForChangeNotificationSubscription [-Period] <Number> 
+```
+
+## DESCRIPTION
+
+This cmdlet supports retrieving the total number of messages sent by a user in chat/channel and whether a user has the required license(s) to sent change notification events when subscribed for chat or channel messages. For more details please review https://docs.microsoft.com/en-us/graph/teams-licenses
```

Comment:
```suggestion
This cmdlet supports retrieving the total number of messages sent by a user in chat/channel and whether a user has the required license(s) to send change notification events when subscribed for chat or channel messages. For more details, please review [Licenses for subscribing to chat messages](/graph/teams-licenses).
```

Reply:
@JohanFreelancer9 - the license information page in different repo , hope relative link will work in that case as well.